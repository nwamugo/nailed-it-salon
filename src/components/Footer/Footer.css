.Footer {
  position: absolute;
  width: 100%;
  height: 130px;
  left: 0px;
  top: 2308px;
  background: black;
}
.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 18px;
  line-height: 21px;
  text-align: center;
  color: white;
}
i {
  font-size: 35px;
  margin-top: 20px;
}
