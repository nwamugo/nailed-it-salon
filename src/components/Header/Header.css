.Header {
  position: absolute;
  width: 100%;
  height: 93px;
  left: 0px;
  top: 0px;
}
.head {
  font-style: normal;
  font-weight: 600;
  font-size: 29px;
  line-height: 29px;
  color: #000000;
  position: absolute;
  height: 29px;
  left: 48px;
  top: 28px;
}
.menu {
  position: absolute;
  height: 14px;
  left: 660px;
  top: 36px;
  font-style: normal;
  font-weight: normal;
  font-size: 12px;
  line-height: 14px;
  /* identical to box height */
  text-align: center;

  color: #000000;
}

a {
  color: black !important;
}
a:hover {
  color: #f0a07f !important;
}
a.nav-link.active {
  color: #f0a07f !important;
}
