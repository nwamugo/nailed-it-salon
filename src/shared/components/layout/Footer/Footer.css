.Footer {
  position: absolute;
  width: 100%;
  height: 8.125rem;
  top: 144.25rem;
  background: black;
}

.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 1.125rem;
  line-height: 1.3125rem;
  text-align: center;
  color: white;
}

.row {
  width: 100%
}

i {
font-size: 2.1875rem;
margin-top: 1.25rem;
  color: white;
}

a.social {
  color: black;
}
