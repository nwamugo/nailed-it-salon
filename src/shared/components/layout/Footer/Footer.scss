.Footer {
  position: absolute;
  width: 100%;
  height: 13rem;
  top: 230rem;
  background: black;
}

.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 1.8rem;
  line-height: 2.1rem;
  text-align: center;
  color: white;
}

.Footer .row {
  width: 100%
}

i {
  font-size: 3.5rem;
  margin-top: 2rem;
  color: white;
}

a.social {
  color: black;
}
