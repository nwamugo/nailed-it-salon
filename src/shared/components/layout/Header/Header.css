.Header {
  position: absolute;
  width: 100%;
  height: 5.8125rem;
}
.head {
  font-style: normal;
  font-weight: 600;
  font-size: 1.8125rem;
  line-height: 1.8125rem;
  color: #000000;
  position: absolute;
  height: 1.8125rem;
  left: 3rem;
  top: 1.75rem;
}
.menu {
  position: absolute;
  height: 0.875rem;
  left: 41.25rem;
  top: 2.25rem;
  font-style: normal;
  font-weight: normal;
  font-size: 0.75rem;
  line-height: 0.875rem;
  /* identical to box height */
  text-align: center;

  color: #000000;
}

a {
  color: black !important;
}
a:hover {
  color: #f0a07f !important;
}
a.nav-link.active {
  color: #f0a07f !important;
}

.u-margin-right {
  margin-right: -12rem;
}
