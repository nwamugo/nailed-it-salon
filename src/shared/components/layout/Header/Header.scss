.nav-logo {
  margin-left: -10px;

  @media (min-width: 320px) and (max-width: 480px) {
    width: 100px;
  }
  @media (min-width: 481px) and (max-width: 767px) {
    width: 120px;
  }
}
.navbar-toggler {
  font-size: 4rem;
  font-weight: bold;
}
.navbar-nav {
  font-style: normal;
  font-weight: normal;
  font-size: 1.8rem;
  line-height: 1.4rem;
  /* identical to box height */
  text-align: center;
  color: #000000;
  margin-left: 45%;

  @media (min-width: 992px) and (max-width: 1280px) {
    margin-left: 37%;
  }
}
.navbar-nav li {
  margin-right: 7px;
}
.navbar-nav li a {
  color: black !important;
}
.navbar-nav a:hover {
  color: #cf6bab !important;
}
.navbar-nav a.nav-link.active {
  color: #cf6bab !important;
}
