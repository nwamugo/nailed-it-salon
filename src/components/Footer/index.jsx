/* eslint-disable no-irregular-whitespace */
import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className="Footer">
      <div className="row">
        <div className="foot col-md-12">
          <i className="fab fa-facebook" />
          <span>&nbsp;&nbsp;&nbsp;</span>
          <i className="fab fa-instagram" />
          <span>&nbsp;&nbsp;&nbsp;</span>
          <i className="fab fa-twitter" />
          <br />
          <br />
          <span>Copyright &copy; 2019Â by Nail Salon</span>
        </div>
      </div>
    </div>
  );
};

export default Footer;
