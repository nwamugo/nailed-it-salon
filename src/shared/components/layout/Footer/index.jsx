/* eslint-disable no-irregular-whitespace */
import React from 'react';
import './Footer.scss';
import NailLogo from '../../../images/nailLogo.png';

const Footer = () => {
  return (
    <div className="Footer">
      <div className="row">
        <div className="foot col-md-12">
          <img src={NailLogo} width="130px" className="foot-logo" alt="logo" />
          <br />
          <a
            href="https://www.snapchat.com/add/n_naileditsa"
            className="social"
            rel="noopener noreferrer"
          >
            <i className="fab fa-snapchat" />
          </a>
          <span>&nbsp;&nbsp;&nbsp;</span>
          <a
            href="https://www.instagram.com/20naileditsa/"
            className="social"
            rel="noopener noreferrer"
          >
            <i className="fab fa-instagram" />
          </a>
          <span>&nbsp;&nbsp;&nbsp;</span>
          <br />
          <br />
          <span className="foot-copyright">Copyright &copy; 2020</span>
        </div>
      </div>
    </div>
  );
};

export default Footer;
