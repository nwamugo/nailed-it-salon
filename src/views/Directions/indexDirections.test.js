import React from 'react';
import { render } from 'react-dom';
import { shallow } from 'enzyme';

import DirectionsPage from './index';
import Header from '../../shared/components/layout/Header';
import Footer from '../../shared/components/layout/Footer';
import Direction from '../../components/Directions';

describe('Directions', () => {
  it('should not regress', () => {
    const component = shallow(<DirectionsPage />);

    expect(component).toMatchSnapshot();
  });

  it('renders the Directions without crashing', () => {
    const div = document.createElement('div');
    render(<Direction />, div);
  });

  it('renders the Header without crashing', () => {
    const div = document.createElement('div');
    render(<Header />, div);
  });

  it('renders the Footer without crashing', () => {
    const div = document.createElement('div');
    render(<Footer />, div);
  });
});
