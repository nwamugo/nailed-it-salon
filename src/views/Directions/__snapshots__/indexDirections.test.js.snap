// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Directions should not regress 1`] = `
<div>
  <Direction />
  <Footer />
</div>
`;
